### PR TITLE
Many Fixes :)

### PR DESCRIPTION
**Pull request by edridudi**

### CODE REVIEW

**Comment by edridudi** [comment]
Not sure about Sonatype's suggested changes.
Looks like we should ignore them.

File: function/src/main/java/com/bloxbean/cardano/client/function/TxOutputBuilder.java [hunk 1]
```diff
@@ -23,6 +23,7 @@ default TxOutputBuilder and(TxOutputBuilder after) {
         };
     }
 
+    @Deprecated
     default TxBuilder buildInputs(TxInputBuilder after) {
```
Comment (satran004) [inline function/src/main/java/com/bloxbean/cardano/client/function/TxOutputBuilder.java:27]: @edridudi Let's not deprecate the current buildInputs(TxBuilderInput), it should just call the new buildInput() with separateChangeOutput = false. So the default behavior is still same. Also, please add some javadoc.

File: function/src/main/java/com/bloxbean/cardano/client/function/TxOutputBuilder.java [hunk 2]
```diff
@@ -23,40 +23,26 @@ default TxOutputBuilder and(TxOutputBuilder after) {
...
-                        txOutputSameAddress.setValue(txOutputSameAddress.getValue().plus(txOutput.getValue()));
-                    } else {
-                        transaction.getBody().getOutputs().add(txOutput);
-                    }
-                });
-
-            //Clear multiasset in the context
-            context.clearMintMultiAssets();
-        };
+        return buildInputs(after, false);
```
Comment (edridudi) [inline function/src/main/java/com/bloxbean/cardano/client/function/TxOutputBuilder.java:34]: it's a mistake here we should send it with true not false.

Comment (satran004) [reply function/src/main/java/com/bloxbean/cardano/client/function/TxOutputBuilder.java:34]: @edridudi I have updated separateChangeOutput to mergeChangeOutput

https://github.com/bloxbean/cardano-client-lib/commit/99e4dd5dbcf6fb57a789b50349dafd5d8d86ce7b

Comment (edridudi) [reply function/src/main/java/com/bloxbean/cardano/client/function/TxOutputBuilder.java:34]: Ok cool

**Pull request merged**